packages/block-library/src/search/index.php

```diff
@@ -102,11 +105,11 @@ function render_block_core_search( $attributes ) {
 		}
 
 		// Include the button element class.
-		$button_classes .= ' ' . WP_Theme_JSON_GUTENBERG::__EXPERIMENTAL_ELEMENT_BUTTON_CLASS_NAME;
-
+		$button_classes[] = WP_Theme_JSON_GUTENBERG::__EXPERIMENTAL_ELEMENT_BUTTON_CLASS_NAME;
+var_dump( $button_classes );
```

Comment:
I know you plan to remove this, but I'm adding a note as an earmark 😄

Reply:
Done